cdap-integration-test/src/main/java/co/cask/cdap/test/IntegrationTestBase.java

```diff
@@ -118,15 +108,18 @@ protected String getAccessToken() {
   }
 
   private void assertIsClear() throws Exception {
+    // TODO: support multiple namespaces
```

Comment:
support multiple namespaces? Isn't it sufficient to assert that the default namespace is the only one existing?